docs/UserGuide.md

```diff
@@ -42,124 +42,125 @@ CCACommander Ultra Promax Xtra 9000PLUS is the one-stop app for CCA Heads to man
 
 Examples:
-* `create /member n/CHU WEI RONG g/Male p/98765432 e/chuweirongrocks@gmail.com a/19 Kent Ridge Crescent, Singapore 119278` creates a member `CHU WEI RONG` in CCACommander.
+* `createMember n/CHU WEI RONG g/Male p/98765432 e/chuweirongrocks@gmail.com a/19 Kent Ridge Crescent, Singapore 119278` creates a member `CHU WEI RONG` in CCACommander.
 
-### Delete a Member : `delete /member`
+### Delete a Member : `deleteMember`
 
 Deletes the member at the specified index.
 
-Format: `delete /member MEMBER_INDEX`
+Format: `deleteMember MEMBER_INDEX`
 
 * Deletes the member at the specified `MEMBER_INDEX`.
 * The index refers to the index number shown in the displayed member list.
 * The index **must be a positive integer** that is within the range of the length of the member list.
 
 Examples:
-* `delete /member 1 ` deletes the 1st member in the member list.
-* `delete /member 10 ` deletes the 10th member in the member list.
+* `deleteMember 1 ` deletes the 1st member in the member list.
+* `deleteMember 10 ` deletes the 10th member in the member list.
 
 ### List all Members and all Events : `list`
 
 List all members and all events in the CCA in 2 separate columns.
 
 Format: `list`
 
-### View Members of Event : `view /event`
+### View Members of Event : `viewEvent`
 
 Lists all the members of a specified event index.
 
-Format: `view /event EVENT_INDEX`
+Format: `viewEvent EVENT_INDEX`
 * Views the members of the event at the specified `EVENT_INDEX`.
 * The index refers to the index number shown in the displayed event list.
 * The index **must be a positive integer** that is within the range of the length of the event list.
 
 Examples:
-* `view /event 1` displays members of the 1st event in the event list.
-* `view /event 10` displays members of the 10th event in the event list.
+* `viewEvent 1` displays members of the 1st event in the event list.
+* `viewEvent 10` displays members of the 10th event in the event list.
 
-### View Events of Member : `view /member`
+### View Events of Member : `viewMember`
 
 Lists all the events of a specified member index.
 
-Format: `view /member MEMBER_INDEX`
+Format: `viewMember MEMBER_INDEX`
 
 * Views the events of the member at the specified `MEMBER_INDEX`.
 * The index refers to the index number shown in the displayed member list.
 * The index **must be a positive integer** that is within the range of the length of the member list.
 
 Examples:
-* `view /member 1` displays members of the 1st member in the member list.
-* `view /member 10` displays members of the 10th member in the member list.
+* `viewMember 1` displays members of the 1st member in the member list.
+* `viewMember 10` displays members of the 10th member in the member list.
 
-### Create an Event : `create /event`
+### Create an Event : `createEvent`
 
 Creates a new event and adds it to the database.
 
-Format: `create /event n/EVENT_NAME [l/LOCATION] [d/DATE]`
+Format: `createEvent n/EVENT_NAME [l/LOCATION] [d/DATE]`
```

Comment:
```suggestion
Format: `createEvent n/EVENT_NAME [l/LOCATION] [d/DATE] [t/TAG]...`
```
The same goes for createEvent as createEvent supports tagging as well.

Reply:
Agreed, will add in the tag component!